app/models/translation.rb

```diff
@@ -112,6 +112,7 @@ def name_desc_onesky
     p = download_translated_phrases('name_description.xml')
     self.translated_name = p['name']
     self.translated_description = p['description']
+    self.translated_tagline = p['tagline']
```

Comment:
Can you add a test for this?

Reply:
There aren't any existing specs for this class that I could see...

Reply:
https://github.com/CruGlobal/mobile-content-api/blob/master/spec/models/translation_spec.rb#L157

Reply:
....
````
Offenses:

spec/models/translation_spec.rb:165:7: C: Example has too many expectations [3/2].
      it 'updates from OneSky' do
      ^^^^^^^^^^^^^^^^^^^^^^^^

130 files inspected, 1 offense detected